extensions/gdx-tools/src/com/badlogic/gdx/tools/etc1/ETC1AtlasCompressionResult.java

```diff
@@ -0,0 +1,59 @@
+

```

Comment:
This is open source software, any contributor can/may review it. Purposely choosing a different license than that of the rest of of the project is being difficult. Not following the contributing guidelines is being difficult. I don't think this is a side-effect of not having an assigned person to review the PR.